### PR TITLE
test(@wallet): fix broken test on wallet

### PR DESCRIPTION
**Pull request by alaibe**

### CODE REVIEW

File: ui/app/AppLayouts/Profile/views/wallet/MainView.qml [hunk 1]
```diff
@@ -106,7 +106,6 @@ Column {
         objectName: "generatedAccounts"
         model: walletStore.generatedAccounts
         delegate: WalletAccountDelegate {
-            objectName: "walletSettingsAccountDelegate"
```
Comment (alaibe) [inline ui/app/AppLayouts/Profile/views/wallet/MainView.qml:109]: @jrainville there is already an objectName set inside the walletAccountDelegate file but more importantly, we cannot set a static object name into a list as it prevent us then to find a specifc one. Doing that broke the delete account test.

**Review by jrainville — APPROVED** [review]
My bad. Thanks for the fix.

**Review by noeliaSD — COMMENTED** [review]
The test fails for me in the following step `When the user adds a saved address named <name> and address <address>` bc the button to add the address is disabled. 

See the video for better understanding: 

https://user-images.githubusercontent.com/97019400/183596982-d85a8bfb-536f-41e2-b61e-1f33eb04eb0e.mov

**Review by noeliaSD — APPROVED** [review]
Approved since it is the expected behaviour due to app updates @igor-sirotin is currently working on.

**Pull request merged**